src/config/shared.js

```diff
@@ -32,3 +32,38 @@ export function extractBucketMetadata(transformedData) {
   etc.bucketSortOrder = sorter;
   return etc;
 }
+
+export const facets = {
+  activeBuckets: {
+    isMulti: true,
+    defaultValue: [],
+  },
+  aggregationLevel: {
+    defaultValue: 'build_id',
+  },
+  channel: {
+    defaultValue: 'nightly',
+  },
+  os: {
+    defaultValue: 'Windows',
```

Comment:
I've always felt the default here should be `*` for all OS's. And if the user wants to drill down they can. What do others think?

Reply:
>     1. can you give more context for having `dimensions.productDimensions` vs. just `dimensions`? It's unclear what the additional nesting accomplishes.

I'm fine removing that. It would make things easier. But I thought we agreed on Slack that we didn't want to flatten it out.

Reply:
I opened an issue for this: #753